### PR TITLE
MessageLogger no longer prints summary by default

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

- The option is still available to turn on in the configuration.
- Always print number of dropped messages if any were dropped.

#### PR validation:

Code compiles and framework unit test pass.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc [hunk 1]
```diff
@@ -885,6 +885,15 @@ namespace edm {
...
           m_statisticsDestControls[i]->wipe();
       }
+      auto dropped = m_tooManyWaitingMessagesCount.load();
+      if (m_statisticsDestControls.empty() and dropped != 0) {
+        if (m_stream_ps.find("cerr") != m_stream_ps.end()) {
+          std::cerr << "MessageLogger: dropped waiting message count " << dropped << "\n";
+        }
+        if (m_stream_ps.find("cout") != m_stream_ps.end()) {
+          std::cout << "MessageLogger: dropped waiting message count " << dropped << "\n";
+        }
```
Comment (makortel) [inline FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc:895]: What if `m_stream_ps` contains both `cerr` and `cout`? Would it still be appropriate to print the message in both?

Comment (Dr15Jones) [reply FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc:895]: If that would be the case (and I don't think I've found a case where that is actually done) maybe they are redirecting output to two different files than both messages could still be useful. But I'm not wedded to the idea.

Comment (Dr15Jones) [reply FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc:895]: @makortel Do you want to have this changed to only output to cerr if both cerr and cout are active or keep it as it is?

Comment (makortel) [reply FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc:895]: Let's leave it as it is.

**Comment by makortel** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**